src/bridge/askpass.c

```diff
@@ -27,7 +27,7 @@
 #include "common/cockpitmemory.h"
 #include "common/cockpittransport.h"
 #if WITH_DEBUG
-#include "common/cockpittest.h"
+#include "common/cockpitstacktrace.h"
```

Comment:
I wonder if that is still actually the right thing to do. systemd-coredump does that these days, in a much nicer way; and Apport on the Ubuntu side. I wouldn't mind if we just ripped out all that stuff entirely -- I can't remember a single situation in the last 5 years where this would have actually helped us. I haven't even *seen* these in bug reports and journals.

Reply:
sehr gerne.